src/rust/engine/options/src/arg_splitter.rs

```diff
@@ -43,13 +66,14 @@ impl ArgSplitter {
             known_goals_map.insert(goal_info.scope_name.to_owned(), goal_info);
         }
 
-        ArgSplitter {
+        Self {
             build_root: build_root.to_owned(),
             known_goals: known_goals_map,
         }
     }
 
-    pub fn split_args(&self, args: Vec<String>) -> SplitArgs {
+    // Split the given args, which must *not* include the argv[0] process name.
```

Comment:
Now that this is called only from the Rust options parser, which itself never sees argv[0]